scripts/build.ps1

```diff
@@ -587,9 +602,32 @@ function Create-NugetPackages
         Set-ScriptFailedOnError
     }
 
+    Verify-Nuget-Packages $packageOutputDir
```

Comment:
Add a comment to specify why this is needed

Reply:
Done.